src/plugins/controls/common/control_group/control_group_panel_diff_system.ts

```diff
@@ -65,6 +67,7 @@ export const ControlPanelDiffSystems: {
         Boolean(singleSelectA) === Boolean(singleSelectB) &&
         Boolean(existsSelectedA) === Boolean(existsSelectedB) &&
         Boolean(runPastTimeoutA) === Boolean(runPastTimeoutB) &&
+        isEqual(searchTechniqueA ?? 'prefix', searchTechniqueB ?? 'prefix') &&
```

Comment:
Nit: Probably, we should have a `DEFAULT_OPTIONS_LIST_SEARCH_TECHNIQUE` variable to make this defaulting logic more obvious.

Reply:
Done in 6601b7f1398986e49b990c100940dc58ab56af4e :+1: